packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```diff
@@ -6,26 +6,24 @@
 %global gem_require_name %{gem_name}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.8.4
-Release: 8%{?dist}
+Version: 1.10.9
+Release: 1%{?dist}
 Summary: Nokogiri (鋸) is an HTML, XML, SAX, and Reader parser
 Group: Development/Languages
 License: MIT
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.1.0
+Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(mini_portile2) >= 2.3.0
-Requires: %{?scl_prefix}rubygem(mini_portile2) < 2.4
+Requires: %{?scl_prefix}rubygem(mini_portile2) >= 2.4.0
+Requires: %{?scl_prefix}rubygem(mini_portile2) < 2.5
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.1.0
+BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.3.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix}rubygem(mini_portile2) >= 2.3.0
-BuildRequires: %{?scl_prefix}rubygem(mini_portile2) < 2.4
-BuildRequires: libxml2-devel
-BuildRequires: libxslt-devel
```

Comment:
These 2 need to be below `# end specfile generated dependencies` since they're manual. They're the reason for the build failure.

Reply:
Added them below the generated dependencies, thanks @ekohl !